index.html

```diff
@@ -62,17 +72,17 @@ <h1>New Entry</h1>
           </div>
         </div>
         <div class="row j-c">
-          <button id="del-button" class="hidden">Delete Entry</button>
+          <button id="del-button" type="button" class="hidden">Delete Entry</button>
           <button id="save-button" type="submit">SAVE</button>
         </div>
       </form>
     </div>
-    <div data-view="entries" id="entries" class="hidden">
+    <div data-view="entries" id="entries" class="">
       <div class="page-name-wrap">
         <h2 class="page-name">Entries</h2>
-        <a href="#" class="new-button">NEW</a>
+        <a href="#new-entry" class="new-button">NEW</a>
```

Comment:
This should just be an `#`. By doing `href="#new-entry"`, it will force the page to jump to an element with this ID. We don't want to force the user to scroll anywhere